pkgs/misc/brightnessctl/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, coreutils }:
+
+stdenv.mkDerivation rec {
+  name = "brightnessctl-${version}";
+  version = "0.3.2";
+
+  src = fetchurl {
+    url = "https://github.com/Hummer12007/brightnessctl/archive/0.3.2.tar.gz";
+    sha256 = "92687d8739b8395cf942287bdb8905a78b428102ab6029680829ab917363e0ab";
+  };
+
+  patchPhase = ''
+    substituteInPlace 90-brightnessctl.rules --replace /bin/ ${coreutils}/bin/
+    substituteInPlace 90-brightnessctl.rules --replace %k '*'
+  '';
+
+  installPhase = ''
+    mkdir -p $out/{bin,share/man/man1,etc/udev/rules.d}
+    install -D brightnessctl $out/bin
+    install -D brightnessctl.1 $out/share/man/man1
+    install -D 90-brightnessctl.rules $out/etc/udev/rules.d
+  '';
+
+  meta = {
+    homepage = "https://github.com/Hummer12007/brightnessctl";
+    license = stdenv.lib.licenses.mit;
```

Comment:
Could you add yourself as a maintainer here?

Reply:
Thanks for the feedback. Ill work on getting them implemented.